app/src/main/java/org/gdg/frisbee/android/arrow/Organizer.java

```diff
@@ -0,0 +1,42 @@
+package org.gdg.frisbee.android.arrow;
+
+import com.google.android.gms.plus.model.people.Person;
+
+public class Organizer {
```

Comment:
My bad, I thought we have. Nevermind

On Tue, Aug 4, 2015, 6:19 AM Michael Prentice <notifications@github.com>
wrote:

> In app/src/main/java/org/gdg/frisbee/android/arrow/Organizer.java
> <https://github.com/gdg-x/frisbee/pull/455#discussion_r36154523>:
>
> > @@ -0,0 +1,42 @@
> > +package org.gdg.frisbee.android.arrow;
> > +
> > +import com.google.android.gms.plus.model.people.Person;
> > +
> > +public class Organizer {
>
> I don't see another Organizer class in the project. Maybe I don't
> understand your point?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gdg-x/frisbee/pull/455/files#r36154523>.
>